README.md

```diff
@@ -0,0 +1,14 @@
+# Cloud IoT API
+
+[![Go Reference](https://pkg.go.dev/badge/cloud.google.com/go/iot.svg)](https://pkg.go.dev/cloud.google.com/go/iot)
```

Comment:
Just to be sure - we're still planning on uploading our own docs, right? I believe there's a jira for that

Reply:
Yes, I'll remove that section for now. I'll change this to link to our docs.